src/20231106_AaveV3Ethereum_AaveFundingUpdates/AaveFundingUpdates.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Aave Funding Updates"
```

Comment:
I would put maybe part 2 or something to not confuse it with the older funding proposal.